test/cmdlineTests/external_function_pointers_taking_calldata_args/output.json

```diff
@@ -0,0 +1 @@
+{"errors":[{"component":"general","formattedMessage":"Cannot import url (\"/projects/contracts.sol\"): File not found. Searched the following locations: \"\".","message":"Cannot import url (\"/projects/contracts.sol\"): File not found. Searched the following locations: \"\".","severity":"error","type":"IOError"},{"component":"general","formattedMessage":"Cannot import url (\"contract.sol\"): File not found. Searched the following locations: \"\".","message":"Cannot import url (\"contract.sol\"): File not found. Searched the following locations: \"\".","severity":"error","type":"IOError"},{"component":"general","formattedMessage":"Cannot import url (\"/tests/test.sol\"): File not found. Searched the following locations: \"\".","message":"Cannot import url (\"/tests/test.sol\"): File not found. Searched the following locations: \"\".","severity":"error","type":"IOError"}],"sources":{}}
```

Comment:
Please add an `args` file with `--pretty-json --json-indent 4` to get this nicely formatted.

See how we do it in [`standard_debug_info_in_yul_and_evm_asm_print_all`](https://github.com/ethereum/solidity/blob/develop/test/cmdlineTests/standard_debug_info_in_yul_and_evm_asm_print_all) for example.